robot/rover/Science/Science.ino

```diff
@@ -743,6 +881,12 @@ float photoChoice(int led) {
   digitalWrite(S3, binary[3]);
 }
 
+/**
+   Tokenizes strings and returns section by index
+
+   Usage: getValue("hello_robotics_world", '_', 2)
+   This will return the string "world"
```

Comment:
ooh, we should consider using this in the other mcu code.

Reply:
:fire: